cypress/integration/unit/search.spec.js

```diff
@@ -46,4 +56,212 @@ context('Search tests', () => {
       .then(() => cy.selectOptionInSelectByText('20'));
     cy.url().should('not.include', 'aantal=20');
   });
+
+  it('Add content for funky searchterms', () => {
+    const testId = `testId=${currentTimestamp()}: `;
+
+    const PLACE = 'Lāna Hawaï eiland';
+    const KIND = 'Ministerraad';
+    cy.createAgenda(KIND, dateToCreateAgenda, PLACE);
+    cy.openAgendaForDate(dateToCreateAgenda);
+
+    const case1TitleShort = `${testId}Cypress search dossier 1`;
+    const type1 = 'Nota';
+    const newSubcase1TitleShort = 'dit is de korte titel for search 🔍 Lāna Hawaï eiland\n\n';
+    const subcase1TitleLong = 'dit is de lange titel for search and searching 🔎 Principiële accénten\n\n';
+    const subcase1Type = 'In voorbereiding';
+    const subcase1Name = 'Principiële goedkeuring m.h.o. op adviesaanvraag';
+
+    const case2TitleShort = `${testId}Cypress search dossier 2`;
+    const type2 = 'Nota';
+    const newSubcase2TitleShort = `${testId} korte titel for batterij`;
+    const subcase2TitleLong = `${testId} lange titel for peerd`;
+    const subcase2Type = 'In voorbereiding';
+    const subcase2Name = 'Principiële goedkeuring m.h.o. op adviesaanvraag';
+
+    cy.createCase(false, case1TitleShort);
+    cy.addSubcase(type1, newSubcase1TitleShort, subcase1TitleLong, subcase1Type, subcase1Name);
+    cy.openSubcase(0);
+    cy.proposeSubcaseForAgenda(dateToCreateAgenda);
```

Comment:
die proposeSubcaseForAgenda kan foutgaan als er teveel agenda's open staan.
We zouden die code moeten aanpassen zodat we die kalender open doen ipv gewoon de dropdown checken
Het veiligste is in de agenda toevoegen van agendaitems.
getest met de full suite ?